TESTING.md

```diff
@@ -58,13 +54,41 @@ $ node_modules\.bin\electron . configs app.cloud
 
 ```
 
-5. Login using a user:
+#### Manual Installation and Configuration of the Morphic Client
 
-```
-$ curl http://localhost:8081/user/carla/proximityTriggered
-```
+You can find the Morphic Installers in our [shared Google Drive](https://drive.google.com/drive/u/1/folders/1nzXsW83qRejup3D_yVYO9t5i4lKN_p5G).  Please note, this link will only work if you are logged in using a Google account that has permission.
+
+Run the installer you want to test.  When installation is finished, you will need to manually configure the client to use your dev cloud.
+
+1. Edit `C:\Program Files (x86)\Morphic\windows\service.json5`
+1. Update the `GPII_CLOUD_URL` setting to point to your dev cloud, i.e. `https://flowmanager.<username>.dev.gcp.gpii.net`
+1. To avoid errors with the self-signed certificates used in a dev cloud, add a new `NODE_TLS_REJECT_UNAUTHORIZED` and set it to `0`. 
+1. Restart, either by:
+   1. Restarting windows
+   1. Opening PowerShell as an administrator and running `restart-service "morphic service"`
```

Comment:
@the-t-in-rtf While working on GPII-4159, I noticed that this command always crashes with
```
PS C:\Users\vagrant> restart-service "morphic service"
restart-service : Service 'morphic service (morphic-service)' cannot be stopped due to the following error: Cannot
open morphic-service service on computer '.'.
At line:1 char:1
+ restart-service "morphic service"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (System.ServiceProcess.ServiceController:ServiceController) [Restart-Service
   ], ServiceCommandException
    + FullyQualifiedErrorId : CouldNotStopService,Microsoft.PowerShell.Commands.RestartServiceCommand
```

Reply:
The command works with the released installer, as long as you are running PowerShell as an administrator.  If you are running PowerShell as a normal user, you will see exactly that message.  Can you confirm whether you're running PowerShell as an admin?

Reply:
@the-t-in-rtf Looks like default vagrant user lacks admin privileges, running as admin solves this indeed, thanks!